src/Microsoft.ML.Data/Training/ITrainerEstimator.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.ML.Runtime.Training
 {
     public interface ITrainerEstimator<out TTransformer, out TPredictor>: IEstimator<TTransformer>
-        where TTransformer: IPredictionTransformer<TPredictor>
+        where TTransformer: IClassicPredictionTransformer<TPredictor>
```

Comment:
>IClassicPredictionTransformer [](start = 28, length = 29)

I believe this change is incorrect, isn't it?

Reply:
No, maybe i named in reverse, but this is the old interface. 

---
In reply to: [218633926](https://github.com/dotnet/machinelearning/pull/912#discussion_r218633926) [](ancestors = 218633926)